spec/requests/drivers_request_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "Drivers", type: :request do
+  describe "GET /index" do
+    it "returns http success" do
+      get "/startup/index"
```

Comment:
this is going to the wrong location.
```suggestion
      get "/driver/index"
```